lib/Models/TableCatalogItem.js

```diff
@@ -83,9 +83,9 @@ var TableCatalogItem = function(terria, url, options) {
      * Gets or sets the opacity (alpha) of the data item, where 0.0 is fully transparent and 1.0 is
      * fully opaque.  This property is observable.
      * @type {Number}
-     * @default 0.6
+     * @default 0.8
      */
-    this.opacity = 0.6;
+    this.opacity = 0.8;
```

Comment:
I just found it looks much nicer, with richer colours, but still readable map text underneath. Eg. for demos I find myself wanting to dial up the opacity to show it off to best advantage. But it's not critical.
We could perhaps leave the TableCatalogItem default as 0.6, but introduce 0.8 on SDMXJSON catalog items. A bit inconsistent, but then it's not a breaking change.